src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1416,21 +1416,23 @@ void TorrentHandle::resume(bool forced)
     m_nativeHandle.resume();
 }
 
-void TorrentHandle::moveStorage(const QString &newPath)
+void TorrentHandle::moveStorage(const QString &newPath, bool overwrite)
 {
     if (isMoveInProgress()) {
         qDebug("enqueue move storage to %s", qPrintable(newPath));
-        m_queuedPath = newPath;
+        m_moveStorageInfo.queuedPath = newPath;
+        m_moveStorageInfo.queuedOverwrite = overwrite;
     }
     else {
         const QString oldPath = nativeActualSavePath();
         if (QDir(oldPath) == QDir(newPath)) return;
 
         qDebug("move storage: %s to %s", qPrintable(oldPath), qPrintable(newPath));
         // Actually move the storage
-        m_nativeHandle.move_storage(newPath.toUtf8().constData());
-        m_oldPath = oldPath;
-        m_newPath = newPath;
+        m_nativeHandle.move_storage(newPath.toUtf8().constData()
+                                    , overwrite ? libt::always_replace_files : libt::dont_replace);
```

Comment:
please add parentheses around the ternary operator `?:`, everything else looks fine.